include/cantera/base/Units.h

```diff
@@ -77,6 +81,50 @@ class Units
 };
 
 
+//! Unit aggregation utility
+/*!
+ *  Provides functions for updating and calculating effective units from a stack
+ *  of unit-exponent pairs. Matching units are aggregated, where a standard unit
+ *  simplifies access when joining exponents. The utility is used in the context
+ *  of effective reaction rate units.
```

Comment:
I like what this class accomplishes. The only thing I wonder is if we should mark it as experimental, to allow us a little flexibility in refining the API on something that's mostly an internal implementation detail.

Reply:
:+1: ... marking as experimental.